frontend/src/assets/images/icon-business.svg

```diff
@@ -1,7 +1,29 @@
-<svg version="1.2" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" overflow="visible" preserveAspectRatio="none" viewBox="0 0 32 32" xml:space="preserve" y="0px" x="0px" id="Layer_1_1588194074809" width="30" height="30"><g transform="translate(1, 1)"><style type="text/css">
-	.st0_1588194074809{fill:none;stroke:#494949;stroke-linecap:round;stroke-linejoin:round;stroke-miterlimit:10;}
```

Comment:
Removing hardcoded styling so that it can be controlled by the component and our own css.